project/src/main/java/com/google/sps/data/ProductSearchLibrary.java

```diff
@@ -22,24 +26,13 @@
         String formattedParent = ProductSearchClient.formatLocationName(projectId, computeRegion);
         // List all the product sets available in the region.
         for (ProductSet productSet : client.listProductSets(formattedParent).iterateAll()) {
-        // Display the product set information
-        System.out.println(String.format("Product set name: %s", productSet.getName()));
-        System.out.println(
-            String.format(
-                "Product set id: %s",
-                productSet.getName().substring(productSet.getName().lastIndexOf('/') + 1)));
-        System.out.println(
-            String.format("Product set display name: %s", productSet.getDisplayName()));
-        System.out.println("Product set index time:");
-        System.out.println(String.format("\tseconds: %s", productSet.getIndexTime().getSeconds()));
-        System.out.println(String.format("\tnanos: %s", productSet.getIndexTime().getNanos()));
-
-        ProductSetItem productSetItem = new ProductSetItem(productSet.getName().substring(productSet.getName().lastIndexOf('/') + 1),
-                                    productSet.getDisplayName());
-        // The set id of a product set is the last directory of a file path and to get this file path
-        // the index of the last backslash symbol is determined and a substring from the index to the end of the file path is made
-        // to get the set id of a product set.
-        productSets.add(productSetItem);
+
+            ProductSetItem productSetItem = new ProductSetItem(productSet.getName().substring(productSet.getName().lastIndexOf('/') + 1),
+                                            productSet.getDisplayName());
+            // The set id of a product set is the last directory of a file path and to get this file path
+            // the index of the last backslash symbol is determined and a substring from the index to the end of the file path is made
+            // to get the set id of a product set.
+            productSets.add(productSetItem);
```

Comment:
Since you also do this part below, you could pull it out into a helper function and call that in both places. That way, if you change this logic it will be reflected in both places.